web-api/openapi/front_office.yaml

```
@@ -107,6 +107,38 @@ paths:
               schema:
                 $ref: '#/components/schemas/UserInteractionError'
 
+  /semantic_search/{document_id}:
+    get:
+      tags:
+        - front office
+      summary: Returns documents similar to the given document.
+      description: |-
+        This endpoints get-queries a virtual collection do documents containing documents
+        semantically similar to the specific documents ranked/ordered from the most to the
+        least similar.
```

We don't need to rerank.
```suggestion
        Returns a list of documents that are semantically similar to the one given as input.
        Each document contains the id, the score and the properties.
        The score is a value between 0 and 1 where a higher value means that the document is more similar to the one in input
```